src/vaishnaviVJun20_Collection/ReplaceString.java

```diff
@@ -0,0 +1,48 @@
+package vaishnaviVJun20_Collection;
+
+import java.util.ArrayList;
+
+/*Program 2 :
+Display on which index Techno is available, Replace all Techno with Credits from given ArrayList, and count total number of Credits.
+["Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"]
+
+Output : 
+Techno word index in given arrayList is 1,3,5 
+["Maulik","Credits" ,"Krishna", "Credits", "Credits", "Credits"]
+Total Credits words :- 4 */
+
+public class ReplaceString {
+
+	void replaceGivenStringInArrayList(ArrayList<String> arrList, String strToReplace, String strReplaceWith) {
+		int count = 0;
+		System.out.print("Techno word is present in a given arrayList at index: ");
+		while (arrList.contains(strToReplace)) {
+			int strIndex = arrList.indexOf(strToReplace);
+			System.out.print(strIndex + ",");
+			arrList.set(strIndex, strReplaceWith);
+		}
+		System.out.println("\nOutput arrayList is: " + arrList);
+
+		for (int index = 0; index < arrList.size(); index++) {
```

Comment:
Here you can handle while loop logic in this for as well, like below, jsut FYI
![image](https://user-images.githubusercontent.com/58640315/88614451-67589680-d0ad-11ea-9f8a-48eb2315b2ba.png)